src/libraries/System.Private.CoreLib/src/System/LocalAppContextSwitches.cs

```diff
@@ -8,6 +8,13 @@ namespace System
 {
     internal static partial class LocalAppContextSwitches
     {
+        private static int s_enableUnsafeUTF7Encoding;
+        public static bool EnableUnsafeUTF7Encoding
+        {
+            [MethodImpl(MethodImplOptions.AggressiveInlining)]
+            get => GetCachedSwitchValue("Switch.System.Text.Encoding.EnableUnsafeUTF7Encoding", ref s_enableUnsafeUTF7Encoding);
```

Comment:
Would it make sense to unify on a single naming convention with these AppContext settings?

I know the ones in this file use the pattern `Switch.<namespace>.<name>`. But we also have others that don't start with `Switch` - for example

https://github.com/dotnet/runtime/blob/bd6cbe3642f51d70839912a6a666e5de747ad581/src/coreclr/src/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.cs#L13-L17

And I'm adding a new one for EventSource:

https://github.com/dotnet/runtime/pull/38129/files#diff-f5b0a60692bde26a95ea15bb86a6bfdbR239

```C#
        private static bool InitializeIsSupported() =>
            AppContext.TryGetSwitch("System.Diagnostics.Tracing.EventSource.IsSupported", out bool isSupported) ? isSupported : true;
```

And we plan on adding one for the `System.SR.UsingResourceKeys`, https://github.com/dotnet/runtime/issues/34057, and a DebugMode setting, https://github.com/dotnet/runtime/pull/37288.

I know we can't rename any setting we've already shipped. But going forward, it would be good to have a standard naming convention for new switches (and possibly creating alternative switch names for the existing non-conforming names).

ASP.NET appears to not use the `Switch.` prefix:
https://github.com/dotnet/aspnetcore/blob/6afd6f40c043f96c8fb1ee06cbebbc1d229efddc/src/Identity/Core/src/SignInManager.cs#L380
https://github.com/dotnet/aspnetcore/blob/6afd6f40c043f96c8fb1ee06cbebbc1d229efddc/src/Http/Routing/src/Matching/DfaMatcherBuilder.cs#L44

Reply:
I agree not using `Switch` prefix. it is not adding any value and also the new switches we are introducing are not using this prefix anymore.